src/arkhamdb/DeckImporterMain.ttslua

```diff
@@ -407,6 +410,51 @@ function handleCustomizableUpgrades(cardList, customizations)
   end
 end
 
+-- Callback function for investigator cards and minicards to set the correct state for alt art
+---@param card Object Card which needs to be set the state for
+---@param loadAltInvestigator String Contains the name of alternative art for the investigator ("normal", "revised" or "promo")
+function loadAltArt(card, loadAltInvestigator)
+  if loadAltInvestigator == "normal" then return end
+
+  -- lookup correct stateId for investigator and alt art state
+  local baseId = string.gsub(JSON.decode(card.getGMNotes()).id, "-m", "")
+
+  local stateIdTable = {}
+  -- Roland Banks
```

Comment:
I don't love the hardcoding here.

Could do this by adding something to the states - maybe Memo, maybe an extra metadata field - with the "revised" or "promo" tag, then inspect the card data to find the correct one.  Would be more durable for future changes, but it's probably not worth it given the limited number of alt arts,